spec.md

```diff
@@ -3278,6 +3287,42 @@ Within a memory declaration, all port names must be unique.
 
 Any modifications to names must preserve the uniqueness of names within a namespace.
 
+# Intrinsics
+
+Intrinsics are expressions and statements and which represent implementation-defined, compiler-provided functionality.
```

Comment:
something wrong with the second `and` in this sentence from an english language perspective

Reply:
Yes, thank you, apologies.  Fixed!